tests/test_output/basic_per_summary.out

```diff
@@ -0,0 +1,14 @@
+==========================================================================================
+Layer (type:depth-idx)                   Param #                   Param %
+==========================================================================================
+SingleInputNet                           --                        0.00
```

Comment:
Let's add a percent sign to all of these?

Also, how should we handle decimal precision? Two decimals look okay here but might not work for large models

Reply:
How about change two decimals to one decimal?

Reply:
Two decimals is fine for now, but in the future users might want to increase the precision. This seems like a good starting point though